src/hacks/Radar.cpp

```diff
@@ -107,6 +108,8 @@ void DrawEntity(int x, int y, CachedEntity *ent)
         {
             if (CE_BYTE(ent, netvar.iLifeState))
                 return; // DEAD. not big surprise.
```

Comment:
```suggestion
            if (ent->m_bAlivePlayer())
                return; // DEAD. not big surprise.
```

Not related to your code, but I still need to fix this